src/index.html

```diff
@@ -1,17 +1,157 @@
-<!doctype html>
+
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>The form</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      method="POST"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <label class="input-row">
+          Surname:
+          <input name="surname" autocomplete="off">
```

Comment:
Task requirements specify that you should use types attributes, even though inputs by default are type='text' mark it like so explicitly

Reply:
Also you forgot about some of these attributes:
![image](https://user-images.githubusercontent.com/87986955/235230362-1eb6d626-aa76-4072-bb25-2429bb0be325.png)